src/routes/RequestQueueRoute.js

```diff
@@ -151,11 +234,23 @@ class RequestQueueRoute extends React.Component {
     const { resources, location } = this.props;
     const request = this.getRequest();
     const requests = this.getRequestsWithDeliveryTypes();
+    const notYetFilledRequests = [];
+    const inProgressRequests = [];
+
+    requests
```

Comment:
this logic should not be in render, let us think where to move it and save it to component state

Reply:
Won't do as discussed